Automattic/wp-calypso
# WooCommerce: Remove disabled store referrers stats

#### Changes proposed in this Pull Request

Since the store in WP.com has been removed (see pbIJXs-Hj-p2), much of the store code was also removed. 

One remaining piece was the store referrers stats section and widgets. They were hidden under a feature flag that was only enabled on development since #21752. It never got to production, and because the feature was sunset, it likely never will. So it's time to clean up that code.

I got to this one while working on #58453. This PR fixes a couple of instances from #58453.

#### Testing instructions

* Verify the removed code is not in use.
* Smoke test `/store/stats/orders/day/:site` where `:site` is a WooCommerce store.
* Verify all tests pass.
